src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -357,12 +357,21 @@ public function provideOgAdminRoutes(OgAdminRoutesEventInterface $event) {
       'description' => 'Manage members',
       'path' => 'members',
       'requirements' => [
-        '_og_user_access_group' => 'administer group',
+        '_og_user_access_group' => 'administer group|manage members',
         // Views module must be enabled.
         '_module_dependencies' => 'views',
       ],
     ];
 
+    $routes_info['add_membership_page'] = [
+      'controller' => '\Drupal\og\Controller\OgAdminMembersController::addPage',
+      'title' => 'Add member',
```

Comment:
@jludwig 
```
og_membership.add:
  route_name: entity.node.og_admin_routes.add_membership_page
  title: 'Add a member'
  appears_on:
    - entity.node.og_admin_routes.members
```
Here we name the same thing with a different title. Should be unified.

Reply:
Updated to "Add a member"